metadata/metadata.txt

```diff
@@ -112,6 +112,12 @@
     "description": "This notebook demonstrates the integration of PreFab into Tidy3D to incorporate advanced fabrication constraints for photonic inverse design..",
     "keywords": "inverse design, adjoint optimization, prefab, grating coupler, photonics, Tidy3D, FDTD",
     "feature_image": "N/A"
+  },  
+    "AllDielectricStructuralColor.ipynb": {
+    "title": "How to build and simulate an all-dielectric structural color device using Tidy3D | FDTD",
+    "description": "How to build and simulate an all-dielectric structural color device using Tidy3D | FDTD.",
+    "keywords": "structural color, all-dielectric, color filter, design, parameter sweep, Tidy3D, FDTD",
```

Comment:
maybe add `design plugin` too. for SEO, in case someone searches "Tidy3d design plugin" on google.